lib/http/retriable/performer.rb

```diff
@@ -23,7 +23,7 @@ class Performer
         IOError
       ].freeze
 
-      RETRIABLE_STATUSES = [500].freeze
+      RETRIABLE_STATUSES = (500...Float::INFINITY).freeze
```

Comment:
Although there are no well known 600+ statuses,  I think this range should be explicit about upper bounds:

```suggestion
      RETRIABLE_STATUSES = (500...600).freeze
```